Solutions/Digital Shadows/Data Connectors/Digital Shadows/DigitalShadowsSearchlight_API_functionApp.json

```diff
@@ -119,7 +119,7 @@
         },
         {
             "title": "2. Import Function App Code",
-            "description": "1. In the newly created Function App, select **Functions** from the navigation menu and click **+ Add**.\n2. Select **Timer Trigger**.\n3. Enter a unique Function **Name** in the New Function field and leave the default cron schedule of every 5 minutes, then click **Create Function**.\n4. Click on the function name and click **Code + Test** from the left pane.\n5. Copy the [Function App Code](<Add GitHub link to Function App code>) and paste into the Function App `run.ps1` editor.\n6. Click **Save**."
+            "description": "1. In the newly created Function App, select **Functions** from the navigation menu and click **+ Add**.\n2. Select **Timer Trigger**.\n3. Enter a unique Function **Name** in the New Function field and leave the default cron schedule of every 5 minutes, then click **Create Function**.\n4. Click on the function name and click **Code + Test** from the left pane.\n5. Copy the [Function App Code](https://github.com/digitalshadows/Azure-Sentinel/tree/digital-shadows-version_2.0.2/Solutions/Digital%20Shadows/Data%20Connectors/Digital%20Shadows/DigitalShadowsConnectorAzureFunction) and paste into the Function App `run.ps1` editor.\n6. Click **Save**."
```

Comment:
Hey @nisha-das-metron, the link added is not redirecting to correct path, by looking to the link added I think the path is given to add "DigitalShadowsConnectorAzureFunction" so got this link with correct path, so can you please add this in PR? https://github.com/digitalshadows/Azure-Sentinel/tree/digital-shadows-version_2.0.2/Solutions/Digital%20Shadows/Data%20Connectors/Digital%20Shadows/DigitalShadowsConnectorAzureFunction)